internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -1,10 +1,53 @@
+UniqueAliasesFunctionRole:
```

Comment:
These three added resources are missing in the integ test in `template-with-imported-certs-observability.yml`